app-text/po4a/po4a-0.69.ebuild

```diff
@@ -28,8 +28,12 @@ RDEPEND="app-text/opensp
 	dev-perl/YAML-Tiny
 	sys-devel/gettext"
 DEPEND="${RDEPEND}"
-BDEPEND="app-text/docbook-xml-dtd:4.1.2
-	app-text/docbook-xsl-stylesheets
+BDEPEND="
+	man? (
```

Comment:
Use conditionals should always go after unconditional deps within a variable. It should also be sorted within the constraint too.

But again, see my comment wrt conditional man building.